canonical-web-and-design/vanilla-framework
# u-min-margin--bottom: ensure it keeps text on baseline grid

## Done

Instead setting margin-bottom to 0, for text elements it sets it to the closest baseline. This is very handy when you want to perfectly center a piece of text inside a container like the maas header for example.

Drive by: Remove pattern specific overrides that were doing the same thing, but with the potential to backfire: e.g. p:last-child inside of a card.

## QA

- Pull code
- Run `./run serve --watch`
- http://192.168.64.7:8101/examples/patterns/card/default/ verify it looks ok
- http://192.168.64.7:8101/examples/utilities/margin-collapse/, verify it looks ok

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]
